parts/linux/cloud-init/artifacts/components.json

```diff
@@ -332,6 +334,7 @@
       "downloadURL": "mcr.microsoft.com/azure-policy/policy-kubernetes-webhook:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "1.7.1",
         "1.7.0",
         "1.5.0"
```

Comment:
can we remove 1.5.0?

Reply:
1.5.0 is still used and will be supported until next year

Please confirm @anlandu

Reply:
1.5 was supposed to be fully switched over to 1.7 in RP release 0827 but that got paused so we still have quite a few clusters on 1.5--will remove ASAP but ideally this PR goes out before 1.5 removal as 1.7.1 patches a CVE